rootfs/deis/gconf.py

```diff
@@ -1,4 +1,6 @@
 import os
+import faulthandler
+faulthandler.enable()
```

Comment:
[`faulthandler`](https://docs.python.org/dev/library/faulthandler.html) is handy. I didn't know that was in Python 3.3+.